rlclientlib/utility/config_helper.cc

```diff
@@ -55,6 +68,8 @@ async_batcher_config get_batcher_config(const configuration &config, const char
   res.send_queue_max_capacity = get_int(config, section, name::SEND_QUEUE_MAX_CAPACITY_KB, 16 * 1024) * 1024;
   res.queue_mode = to_queue_mode_enum(get_str(config, section, name::QUEUE_MODE, value::QUEUE_MODE_DROP));
   res.batch_content_encoding = config.get_bool(section, name::USE_DEDUP, false) ? value::CONTENT_ENCODING_DEDUP : value::CONTENT_ENCODING_IDENTITY;
+  // TODO: what should happen if the rate is not (0, 1]?
```

Comment:
should we maybe put a warning/throw something here until we figure out what happens if the rate is not (0,1]?